src/background/index.ts

```diff
@@ -163,18 +163,19 @@ api.runtime.onMessageExternal.addListener((m, s, r) => {
 					name: reg_request.name,
 					state: IntegrationStateDisabled,
 				};
-				api.storage.local.set({
-					integrations,
-					[EventKey]: {
-						type: MessageEvent,
-						message: `<p>The extension <b>${
-							reg_request.name
-						}</b> would like to integrate with BlueBlocker.<br>Visit the <a href="${api.runtime.getURL(
-							'/src/pages/integrations/index.html',
-						)}" target="_blank">integrations page</a> to complete set up.</p>`,
-						options: { html: true },
-					},
-				});
+				// TODO: eliminate html here, cannot generate elements directly as it's not the same document. must be json-serializable
```

Comment:
I'm kinda leery about leaving code commented out; maybe hide it behind a dummy condition that will never be true, or just remove it entirely?

Reply:
I want it to be removed on build. I also want to add a way to make this work since we can't just use the `document.createElement` here

Reply:
ok